static/usr/lib/systemd/system-generators/bootchart

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+set -eu
+
+[ -d /run/bootchart-debug ] ||  mkdir -p /run/bootchart-debug
+
+case "${SYSTEMD_ARCHITECTURE-}" in
+    riscv64)
+        exit 0
+  ;;
+esac
+
+if ! /usr/libexec/core/get-arg core.bootchart >/dev/null; then
+    exit 0
+fi
+
+# /var/lib/snapd is bind mounted by a unit from fstab-generator
+# It is not mounted yet.
+modeenv=/run/mnt/data/system-data/var/lib/snapd/modeenv
+mode="$(/usr/libexec/core/get-mode mode "${modeenv}")" || mode="unknown"
+
+if [ "${mode}" != run ]; then
```

Comment:
My mistake. I am testing now. But I have found some issues with the stop script and install mode. I will try to fix that as well.

Reply:
I have rewritten the stop script to write in the correct place. Also fixed the dependencies so that the data partition is not unmounted before. Now it is work. I get can get the logs from the installation boot.